### DISCUSSION TITLE
Units in Allowed Amount File

### DISCUSSION THREAD

**Discussion by KMoriarity** [CMSgov/price-transparency-guide#263] [Q&A]
For the allowed amount file, should we be reporting a per unit submitted charge and per unit allowed or the total charged/allowed for the billing code, ignoring the units. 

For example, if we have the following 2 claims from a provider for 26615.
One for 2 units with a charge of 200 and an allowed of 100 and one with 1 unit with a charge of 100 and an allowed of 50.  

Should the file have this just this record with the per unit financials
26615 charge: 100 Allowed: 50

Or both records with the total financials
26615 charge: 200 Allowed: 100
26615 charge: 100 Allowed: 50

Similarly, should we be reporting per unit amounts for anesthesia services?

**Comment by angelamarie0975** [comment]
Our solution to this was to do a calculation of the allowed amount and billed charge divided by the units as long as the units are greater than 1. If there are still multiple rates after reducing the amounts to the per unit amount, then we are reporting each of the different amounts in the file as long as there are 20+ claims for those values. If the units are less than 1, we were instructed by our legal team to just report all the values as long as there are 20+ claims as it was not as easy to do the calculation in the reverse and it add up correctly to the same per unit amount because of the precision - this typically applied to the anesthesia and injectable services. I am not sure what others are doing for this though.
